### ISSUE TITLE
Hope the local language config

### ISSUE THREAD

**Issue by archonwang** [dnoise/Redmine-Wordkload-Dnoise#2]
It seems that the local language cannot be configged. Could it apply the new future?

**Comment by speedy32129** [comment]
Same issue here.  We prefer just to see the english only labels.  I'm hoping they are going to fix this!  It's much easier with translations files and the community will be more than happy to help you with them. 

Also and FYI for everyone so far this plugin is working on Chiliproject too!

**Comment by dnoise** [comment]
Hi :)

Thanks for the FB

This is a develop made for internal use in our agency, but the result was good enough to release it. We will try to make it "translation friendly"... but not today, even not tomorow ;)... our workload doesnt allow us to work more hours in this project for a while.

We encourage you to make all the implementations you can and we promise we will upload all the changes we made in the plug

;)

**Comment by speedy32129** [comment]
I have the updated view and English and Spanish language files completed.  The only thing i haven't quite figured out is how to update the calendar language in the javascript file.  Now how can I get the zip file containing the changes to you?

**Comment by cagnulein** [comment]
@speedy32129 why don't you fork the project? i'd really appreciate that...i could translate it in italian too.

**Comment by speedy32129** [comment]
@cagnulein, 

I really think the mods I made so far have been pretty simple.  I'd prefer to keep it all together.  However if @dnoise doesn't want my files then I probably will fork the project.  But I do admit I'm very new ro ruby and to Redmine but I think I'm picking up fairly fast.  My next mod is to switch the calendar to use the redmine calendar since it's already completed for multiple languages.  Then I plan on working on the ajax forms for translation as well.  If any one want's my changes I will post them for you to download from my website.

Let me know if your interested both the author dnoise and users!

Thanks,

speedy32129

**Comment by cagnulein** [comment]
dnoise is still asleep :)
Forking a project on github doesn't allow to merging together in future? If it's so, i think it should be the best way.

**Comment by cforce** [comment]
Of course, merging is one of gits mail featuresds. Did you fork? I am looking forward

**Comment by dnoise** [comment]
Hi!... we are on hollidays :) we will back next week

**Comment by speedy32129** [comment]
my preliminary fork is complete.  Without completely tearing it apart.  I plan on making a few more updates.  Go to https://github.com/speedy32129/Workload-Toucan-Soft.  Hopefully they will merge my changes.  If not I'm adding my ideas and off we go. Enjoy!

**Comment by dnoise** [comment]
Sory!!! we have a big delay with project due to DNOiSE´s workload...  we will check it up ASAP. 

Aiss, really, sory
